src/tls13.c

```diff
@@ -8510,7 +8511,7 @@ static int SendTls13Certificate(WOLFSSL* ssl)
         listSz = 0;
     }
     else {
-        if (!ssl->buffers.certificate) {
+        if (!ssl->buffers.certificate || !ssl->buffers.certificate->buffer) {
```

Comment:
`ssl->buffers.certificate->buffer` is only dereferenced if `certSz > 0`.  I can imagine there's code somewhere relying on null `buffer` being tolerated when `ssl->buffers.certificate->length` is zero, so this nullness check should only be done when `ssl->buffers.certificate->length > 0`.

Reply:
I agree its a bit redundant, but is consistent with the rest of the code. Search for `!ssl->buffers.certificate` and you will see every single check in internal.c, ssl.c and tls13.c use `if (!ssl->buffers.certificate || !ssl->buffers.certificate->buffer)`

If we want to address that redundancy check separately I am on board, but I think at least for this PR its best to have the checks be consistent.